packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,278 @@
+# Terra Application Layout
+
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
+- Additionally, the ApplicationLayout requires the presence of any `Router` component (provided by `react-router-dom`) in its parent hierarchy.
+
+## Props
+
+### `nameConfig`
+
```

Comment:
Per [the comment here](https://github.com/cerner/terra-framework/pull/116/files#r176071517), I'd add a line after the prop name heading to note that it is required. Something like

Is required: **true**